### PR TITLE
Add Gnosis to scaffold-eth-2 configurations

### PR DESCRIPTION
**Pull request by 0xMako**
## Description

_Concise description of proposed changes, We recommend using screenshots and videos for better description_

## Additional Information

- [ ] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [ ] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #{issue number}_

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address:

### CODE REVIEW

File: packages/hardhat/hardhat.config.ts [hunk 1]
```diff
@@ -67,6 +67,20 @@ const config: HardhatUserConfig = {
...
+        authorization: `Bearer ${process.env.GATEWAY_API_KEY}`,
+      },
+      accounts: [deployerPrivateKey],
+    },
+    gnosisChiado: {
+      url: `https://rpc.eu-central-2.gateway.fm/v4/gnosis/archival/chiado`,
+      httpHeaders: {
+        authorization: `Bearer ${process.env.GATEWAY_API_KEY}`,
+      },
+      accounts: [deployerPrivateKey],
```
Comment (carletex) [inline packages/hardhat/hardhat.config.ts:82]: I'd use the official RPCs listed on https://docs.gnosischain.com/tools/rpc/#gnosis

https://rpc.gnosischain.com && https://rpc.chiadochain.net

**Review by carletex — CHANGES_REQUESTED** [review]
Thanks @0xMako !

Left a couple of comments

File: packages/hardhat/hardhat.config.ts (hunk 1 above)
Comment (0xMako) [reply packages/hardhat/hardhat.config.ts:82]: FYI - Gnosis will be sunsetting the "official" RPCs in favor of an ecosystem approach. Gateway will be taking over the "public" RPC. 

https://forum.gnosis.io/t/gip-70-should-gnosisdao-pass-a-grant-proposal-to-partner-with-gateway-fm/6079

**Comment by 0xMako** [comment]
@carletex Oh gosh - I didn't mean to do a PR to the main `scaffold-eth-2` repo, this was a quick demo that I was using to show someone. So sorry about this! 

I'll submit another PR in a few days, which will be a bit more cleaned up.

**Pull request closed without merging**